evap/grades/views.py

```diff
@@ -171,7 +171,7 @@ def edit_grades(request, semester_id, course_id, grade_document_id):
     if form.is_valid():
         form.save(modifying_user=request.user)
         messages.success(request, _("Successfully updated grades."))
-        return redirect("grades:course_view", semester.id, course.id)
+        return redirect("grades:course_view", course.id)
```

Comment:
codecov says this is not covered -- do we not have a test for this? I think we should.

Reply:
Yeah, I saw that these two were left, for me it felt a bit extra to test the two ifs, but I don't tend towards either side very strongly. If you tend towards writing a test, I can do that before merging

Reply:
I think a test that submitting the form with correct data actually changes something in the database is appropriate.